pyproject.toml

```diff
@@ -15,6 +15,12 @@ packages = []
 [tool.poetry.scripts]
 notify-api = ""
 
+[tool.ruff]
+ignore = ["D101", "D102", "D103"]
```

Comment:
This removes warnings about undocumented classes. I can remove it if developers prefer to keep it but I don't see it as a common practice at the moment within the code.